types/src/grants.d.ts

```diff
@@ -18,6 +19,25 @@ export type GrantMetadata = {
   name: string;
   description: string;
   logoURI?: string;
+  quote?: string;
+  lastUpdated?: string;
+  projectWebsite?: string;
+  projectGithub?: string;
+  twitterHandle?: string;
 };
 export type GrantMetadataStatus = 'resolved' | 'pending' | 'error';
 export type GrantMetadataResolution = Partial<GrantMetadata> & { status: GrantMetadataStatus };
+
+// Details of a grant in a round
+export type GrantsRoundDetails = {
+  address: string;
+  metaPtr: string;
+  name: string;
+  matchingToken: TokenInfo;
+  donationToken: TokenInfo;
+  contributions: (false | Event)[];
+  matching: number;
+  prediction10: number;
```

Comment:
for 0th prediction point, we assume it to be the initial matching balance correct?

Reply:
Yes thats rights, the 0th prediction is the `matching` value - which is taken from `predictedGrantMatch` rather than the `predictionDiff` 👍